dashboards/k8s-storage-resources-glusterfs-pv.libsonnet

```diff
@@ -0,0 +1,272 @@
+local g = import 'grafana-builder/grafana.libsonnet';
+
+{
+  grafanaDashboards+:: {
+    'k8s-storage-resources-glusterfs-pv.json':
+    local tableStyles = {
+        fop: {
+          alias: 'File Operarions',
+        },
+      };
+    local statPanelArgs = {
+      height: '75px',
+      span : 3,
+    };
+    local brickStatPanelArgs = {
+      height: '75px',
+      span : 4,
+    };
+    local graphPanelArgs ={
+      span :6,
+    };
+      g.dashboard(
+        'K8s / Storage Resources / Persistent Volume / glusterfs',
+        uid=($._config.grafanaDashboardIDs['k8s-storage-resources-glusterfs-pv.json']),
```

Comment:
This has to present here. Dynamic addition of UID, will be taken care by defaults.libsonnet

Reply:
> This has to present here since a stable ID is required at runtime to generate dashboard. 
>
I don't think it is required here.
>
>Dynamic addition of UID, will be taken care by defaults.libsonnet
>
AFAIK, It will create new UID and attach it to each dashboards at runtime, before the dashboard objects get parsed.